### PR TITLE
Hp/edit events

### PR DESCRIPTION
**Pull request by CN-Discretemathematics**

### CODE REVIEW

File: src/components/EditEventComponent/EditEvent.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      content,
+      location,
+      attendees,
+      travelTime,
+      start: start.toDate(),
+      end: end.toDate(),
+      isAllDay
+    };
+
+    try {
```
Comment (EnguerrandINSA) [inline src/components/EditEventComponent/EditEvent.js:48]: Why is that here ? We already have a update method in the eventServices.js file

File: src/components/EventDetailsComponent/EventDetails.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React from "react";
-import moment from "moment";
+import moment from "moment-timezone";
 import "./EventDetails.css";
 
-function EventDetails({ event, onDelete }) {
+moment.tz.setDefault('Europe/Stockholm');
```
Comment (EnguerrandINSA) [inline src/components/EventDetailsComponent/EventDetails.js:5]: Why did you put this in every file ? What is the point of this ? And can't you just define it once ?

Comment (CN-Discretemathematics) [reply src/components/EventDetailsComponent/EventDetails.js:5]: Its purpose is to synchronize time zones, ensuring that the time used in the calendar matches the local time, thus avoiding the issue of displaying incorrect times when editing.i don't know if it is  ok to just define once,so i add them in every file.

File: src/components/EditEventComponent/EditEvent.js (hunk 1 above)
Comment (CN-Discretemathematics) [reply src/components/EditEventComponent/EditEvent.js:48]: The purpose of doing this is to allow for modifications to whether it is an all-day event when using the editing function, ensuring the page remains synchronized.
